tests/utils/test_image_utils.py

```diff
@@ -141,7 +141,7 @@ def test_exif_lookup(self):
         self.assertEqual(sensor_width, 36)
         # Catch lookup failure
         with self.assertRaises(LookupError):
-            db.lookup(make="Canon", model="Canon EOS REBEL T1i")
```

Comment:
Just curious, did we update the database here, that makes the Rebel T1i found, but not the Rebel XT?

Reply:
The database stays the same. The `Rebel T1i` is found because of the newly-added substring matching logic. It is stored as `Canon EOS Rebel T1i / 500D`. However `Rebel XT` from Alamo dataset is not in the database at all.